r/R/dplyr-datetime-helpers.R

```diff
@@ -240,25 +206,29 @@ build_formats <- function(orders) {
 #' @noRd
 build_format_from_order <- function(order) {
   char_list <- list(
+    "T" = "%H-%M-%S",
```

Comment:
Probably a swap of `"T"` to "HMS" should happen in `build_formats` and not here.

Reply:
I'm ok either way. I've moved things around a bit, do you still think so? If yes I'll move it.